content/en/logs/archives/s3.md

```diff
@@ -96,17 +55,53 @@ Next, grant Datadog permissions to write log archives to your S3 bucket. You can
 }
 ```
 
-3. Go to your [Pipelines page in Datadog][5], and select the **Add archive** option at the bottom. Only Datadog users with admin status can complete this and the following step.
+3. Go to your [Pipelines page in Datadog][4], and select the **Add archive** option at the bottom. Only Datadog users with admin status can complete this and the following step.
 
 4. Select the appropriate AWS account + role combination for your S3 bucket. Input your bucket name. Optionally input a prefix directory for all the content of your log archives. Save your archive, and you are finished. 
 
     {{< img src="logs/archives/log_archives_s3_datadog_settings_role_delegation.png" alt="Set your S3 bucket info in Datadog" responsive="true" style="width:75%;">}}
     
-[1]: /help
-[2]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
-[3]: /integrations/amazon_web_services/?tab=allpermissions#installation
-[4]: /logs/archives/rehydrating
-[5]: https://app.datadoghq.com/logs/pipelines
+[1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
+[2]: /integrations/amazon_web_services/?tab=allpermissions#installation
+[3]: /logs/archives/rehydrating
+[4]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+
+{{% tab "EU Site" %}}
+
+1. Modify your bucket to grant write-only access to the Datadog AWS user. Do this by editing your bucket's **permissions**, and setting the **bucket policy** with the following content (replace `<MY_BUCKET_NAME>` with the name of your bucket):
+
+    ```
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "AllowDatadogArchivesUploader",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "464622532012"
+                },
+                "Action": [
+                    "s3:PutObject",
+                    "s3:PutObjectAcl",
+                    "s3:ListMultipartUploadParts",
+                    "s3:AbortMultipartUpload"
+                ],
+                "Resource": "arn:aws:s3:::<MY_BUCKET_NAME>/*"
+            }
+        ]
+    }
+    ```
+
+    {{< img src="logs/archives/log_archives_s3_iam_policy.png" alt="IAM Policy for S3 Archives" responsive="true" style="width:75%;">}}
```

Comment:
There's a current bug with our site where images within tabs, within ordered lists will break the numbering. If this screenshoot is unnecessary, I suggest removing it.

Reply:
removing the screenshot or code bracket in this case is not an option. since this issue is already there, i'm ok with it staying.